location.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./global.css">
+    <link rel="stylesheet" href="./location.css">
+    <title>Location</title>
+</head>
+<body>
+
+    <header class="box">
+        <h1>The Bakery</h1>
+       <!-- Nav / Header Component Here-->
+        <nav class="menu">
+            <a href="index.html">Home</a>
+            <a href="location.html">Location</a>
+            
+        </nav>
+    </header>
+
+    <main class="box">
+        <!-- custom page content here -->
+        <h2>Location</h1>
+
+        <!--Contact information-->
+        <ul>
+
+            <li>Barnes' Bakery</li> <!--Name-->
+            <li>1111 E. Main Street </li> <!--Address-->
+            <li>Durham, NC, 27701</li> <!--Address-->
+            <li>Phone Number: (919) 888-8888</li> <!--Phone number-->
+            <li>BarnesBakery@gmail.com</li> <!--Email address-->
```

Comment:
Please convert this into a link. Inside the li element, add <a href="mailto:barnesbakery@gmail.com">BarnesBarkery@gmail.com</a> for example @KwaiJarrell

Reply:
I have changed the email into a link.